script.js

```diff
@@ -1,60 +1,63 @@
-function Books(id, title, author) {
-   this.id = id;
-   this.title = title;
-   this.author = author;
+class Books {
```

Comment:
Hey @RiyaBulia12, though you have used class-based code, however, the main functionality is still outside the class. Functionality for adding and removing books should be in the class not outside it.  Also, the function to display the list of books should belong to the class. Consider updating your code accordingly.